.travis/osx..script.sh

```diff
@@ -5,4 +5,4 @@ if [ $QT5 ]; then
         export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
 fi
 
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWANT_QT5=$QT5 -DUSE_WERROR=OFF ..
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWANT_QT5=$QT5 -DUSE_WERROR=ON ..
```

Comment:
@jasp00 thanks I didn't realize this was our own flag.  This was added and only ever enabled for Mac travis builds.  My guess is @Lukas-W did this just to get Travis to pass for the time being.  If you've fixed all the clang warnings, is it needed anymore?  https://github.com/LMMS/lmms/commit/9cc1a5931c34535436a2fb82e51031121b3a53d1